src/uglify/minify.js

```diff
@@ -40,7 +40,7 @@ const buildComments = (options, uglifyOptions, extractedComments) => {
   // /^\**!|@preserve|@license|@cc_on/
   if (typeof options.extractComments === 'boolean') {
     condition.preserve = commentsOpts;
-    condition.extract = /^\**!|@preserve|@license|@cc_on/;
+    condition.extract = /^\**!|Copyright|LICENSE|@preserve|@license|@cc_on/ig;
```

Comment:
Are these `uglify-es` defaults aswell ?

Reply:
Not sure, this is the config we've been using at Zocdoc that extracts more of the relevant comments.

Reply:
@evan-scott-zocdoc maybe also do PR in uglify-es?

Reply:
I could, but you're already defining your own override here so me changing it in uglify-es wouldn't fix anything?

Reply:
@evan-scott-zocdoc First in uglify, second we merge here

Reply:
If I remember right for `options.extractComments` which is a feature of `uglifyjs-webpack-plugin`, we took those defaults from `uglify-es` (default preserved comments). It's better to keep `uglifyjs-webpack-plugin` as near to `uglify-es` defaults as possible even for a small change like this.